.circleci/config.yml

```diff
@@ -109,47 +107,34 @@ jobs:
             mv client mattermost/client
             tar -zcf mattermost-enterprise-linux-amd64.tar.gz mattermost/
       - store_artifacts:
-          path: ~/mattermost-enterprise-linux-amd64.tar.gz
-      - run:
-          name: Upload to S3
-          command: |
-            if [[ -z "${CIRCLE_PULL_REQUEST:-}" ]]; then
-              echo "Not a PR using the branch name ${CIRCLE_BRANCH}..."
-              export FOLDER_NAME=${CIRCLE_BRANCH}
-              echo "${FOLDER_NAME}"
-            else
-              echo "This is a PR ${CIRCLE_BRANCH}..."
-              export FOLDER_NAME=$(echo "${CIRCLE_BRANCH}" | sed 's/pull\//PR-/g')
-              echo "${FOLDER_NAME}"
-            fi
-            curl --request POST \
-            --url "https://lambdas.devops.mattermost.com/circleci/uploader?token=${UPLOADER_TOKEN}&vcs-type=github&username=${CIRCLE_PROJECT_USERNAME}&project=${CIRCLE_PROJECT_REPONAME}&build_num=${CIRCLE_BUILD_NUM}&bucket=releases.mattermost.com/${CIRCLE_PROJECT_REPONAME}/commit/${CIRCLE_SHA1}/"
-      - add_ssh_keys:
-          fingerprints:
-            - "1f:ea:9e:d2:11:90:f7:35:0f:86:0d:45:e2:a4:73:f7"
+          path: /home/circleci/project/mattermost-enterprise-linux-amd64.tar.gz
+      - aws-s3/copy:
+          from: /home/circleci/project/mattermost-enterprise-linux-amd64.tar.gz
+          to: s3://releases.mattermost.com/${CIRCLE_PROJECT_REPONAME}/commit/${CIRCLE_SHA1}/
+          arguments: --cache-control no-cache
+
+  build-docker:
+    working_directory: ~/
+    docker:
+      - image: circleci/buildpack-deps:stretch
+    steps:
+      - attach_workspace:
+          at: .
+      - setup_remote_docker
       - run:
           command: |
-            if [[ -z "${CIRCLE_PULL_REQUEST:-}" ]]; then
-              echo "Not a PR using the branch name ${CIRCLE_BRANCH}..."
-              export FOLDER_NAME=${CIRCLE_BRANCH}
-              echo "${FOLDER_NAME}"
-            else
-              echo "This is a PR ${CIRCLE_BRANCH}..."
-              echo "Commit hash ${TAG}"
-              export FOLDER_NAME=$(echo "${CIRCLE_BRANCH}" | sed 's/pull\//PR-/g')
-              echo "${FOLDER_NAME}"
-            fi
             export TAG="${CIRCLE_SHA1:0:7}"
 
             mkdir -p ~/.ssh/
             echo -e "Host github.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config
-            git clone git@github.com:mattermost/mattermost-docker.git
+            git clone git@github.com:mattermost/mattermost-server.git
```

Comment:
I'm uncertain why we checkout the mattermost server here -- can you clarify?

Reply:
because we are building the docker image for mattermost and the dockerfile is located in the mattermost-server repository before it was located in the mattermost-docker but we are moving out from mm-docker to mm-server